livehtml.sh

```diff
@@ -2,4 +2,4 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
 
-sphinx-autobuild -b html doc doc/html --host 0.0.0.0
\ No newline at end of file
+sphinx-autobuild -b html doc doc/html --host 0.0.0.0 --port 8080
```

Comment:
Does VSCode Remote automatically forward this connection for us? If not, should we change the host or echo it so that it will be forwarded?

Reply:
It does: 
![image](https://user-images.githubusercontent.com/42961061/97900772-0e933c80-1d33-11eb-84d2-d022763cf27a.png)

I've updated this port as it clashed with the sandbox.

Reply:
If it does, then let’s change it to localhost rather than 0.0.0.0